gateway/src/clickhouse.rs

```diff
@@ -143,6 +144,7 @@ impl ClickHouseConnectionInfo {
         }
     }
 
+    #[instrument(skip(self))]
     pub async fn run_query(&self, query: String) -> Result<String, Error> {
```

Comment:
We shouldn't log this function as it could contain sensitive data in the query. Either skip_all or remove.